docs/hpc/containers.md

```diff
@@ -30,11 +31,13 @@ To build your own singularity container for use in the shared HPC, you will need
 
 To load this OS onto your VM box, follow along with this [guided tutorial](https://www.youtube.com/watch?v=hE2eOLx0gNU).
 
+If you are logged in to the HPC these should be installed already.
+
 ### Build
 
-Once you have your RedHat OS successfully installed, you will utilize the following "recipe" file template to build an R or Python container.
+Once you have your RedHat OS successfully installed, you will utilize the following "recipe" file template to build a Python container.
 
-Copy and paste the text below into a file called `container.recipe`.
+Copy and paste the text below into a file called `<container-name>.recipe`.
```

Comment:
I think we discussed conventions for container names, right?  If so, could you please add to our etiquette/naming-conventions page?